core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -24,6 +24,9 @@ class TestCaseWriter {
 
     private var counter = 0
     private var usedObjects = UsedObjects()
+    private var previous_chained = false
+    private var previous_id = ""
+    private var chained = false
```

Comment:
better to call them `previousChained` and `previousId`. `_` should rather be used for constants in uppercase, eg `PREVIOUS_ID`

Reply:
Done.